proposals/dao/v2Phase1-part3.ts

```diff
@@ -0,0 +1,117 @@
+import { BalancerLBPSwapper, Core, Tribe, TribeReserveStabilizer } from '@custom-types/contracts';
+import { RunUpgradeFunc, SetupUpgradeFunc, TeardownUpgradeFunc, ValidateUpgradeFunc } from '@custom-types/types';
+import '@nomiclabs/hardhat-ethers';
+import chai, { expect } from 'chai';
+import CBN from 'chai-bn';
+import hre, { ethers } from 'hardhat';
+
+const toBN = ethers.BigNumber.from;
+
+before(() => {
+  chai.use(CBN(ethers.BigNumber));
+});
+
+/*
+
+V2 Phase 1 Upgrade
+
+Part 3 - Deploys collateralization oracle keeper, collateralization oracle guardian, chainlink oracle wrapper for tribe-eth, chainlink composite oracle for tribe-usd, tribe reserve stabilizer, 
+         tribe splitter, pcv equity minter, and the balancer-v2 liquidity bootstrapping pool (and associated swapper). Grants
+         burner role[remove] to the tribe reserve stabilizer, and minter roles to the pvc equity minter & collateralization oracle keeper.
+         Also grants tribe-minter role to the tribe reserve stabilizer, and seeds tribe to the liquidity bootstrapping pool swapper.
+
+----- PART 3 -----
+
+DEPLOY ACTIONS:
+1. Collateralization Ratio Oracle Keeer
+2. Collateralization Oracle Guardian
+3. Chainlink Tribe ETH Oracle Wrapper
+4. Chainlink Tribe USD Composite Oracle
+5. Tribe Reserve Stabilizer
+6. Tribe Splitter
+7. Fei Tribe LBP Swapper
+8. Fei Tribe LBP (Liquidity Bootstrapping Pool)
+9. PCV Equity Minter
+
+DAO ACTIONS:
+1. Grant Burner role to new TribeReserveStabilizer
+2. Grant Minter role to PCV Equity Minter
+3. Grant Minter role to Collateralization Oracle Keeper
+4. Grant Tribe Minter role to Tribe Reserve Stabilizer
+5. Grant Oracle Admin role to Collateralization Oracle Guardian
+6. Seed TRIBE to LBP Swapper
```

Comment:
I'm only seeing step 4 and 6 in the run script

Reply:
Yes, need to sync with you on the oracle-admin stuff.